public/locales/es/translations.json

```
@@ -0,0 +1,49 @@
+{
+	"root.description": "Una alternativa al ecosistema de Nix",
+
+	"header.community": "Comunidad",
+
+	"goals.title": "Objetivos",
+	"goals.independent.title": "Independiente",
+	"goals.independent": "Nosotros forkearemos y mantendremos Nix, NixPkgs y NixOS. Soporte adicional para proyectos como Nix Darwin y Home Manager será considerado.",
```

```suggestion
	"goals.independent": "Mantendremos una bifurcación ("fork") de Nix, NixPkgs y NixOS. Se considerará soporte adicional para proyectos como Nix Darwin y Home Manager.",
```

Looking at some websites like https://git-scm.com/book/es/v2/GitHub-Participando-en-Proyectos fork seems to be translated as "bifurcación". "Forkearemos" doesn't sound so good to me, so we can change it slightly to avoid using a verb. This is just a suggestion, if you don't like having "bifurcación (fork)" in there we can simplify it to just use one of the two.

I also made a small change in the second sentence, and remove "nosotros" because I'd like to avoid gendered language as far as possible, I know it's hard in Spanish but if we can that would be great.

---

"Forkearemos" was definetly something provisional that sounded awful. :skull: "Bifurcación" on the other hand fits nicely.

Also thanks resolving the gendered language, I missed that one.